### PR TITLE
44 new input data

### PR DESCRIPTION
**Pull request by jrmiro**

### CODE REVIEW

File: pymica_dev_jr.yml
Comment (ecasellas) [inline pymica_dev_jr.yml:1]: This file should not be included in the PR.

File: pymica_tests/pymica2_test.py [hunk 1]
```diff
@@ -380,18 +380,28 @@ def test_init_interpolate_id2d(self):
...
+                      'dist': 0.8583929293407604},
+                     {'id': 'C7', 'value': 5.6, 'altitude': 427,
+                      'lon': 1.16234, 'lat': 41.66695,
+                      'dist': 0.8387222708681318}]
+        
+        
+
         id2d = PyMica('id2d', 'pymica_tests/data/config_test.json')
-        field = id2d.interpolate('pymica_tests/data/init_data.json')
+        field = id2d.interpolate(data_dict)
```
Comment (ecasellas) [inline pymica_tests/pymica2_test.py:400]: Why the result values have changed? The methodology is the same, the result, I think, it should be the same.

File: pymica_tests/pymica2_test.py [hunk 2]
```diff
@@ -409,13 +419,22 @@ def test_init_interpolate_id3d(self):
...
+                      'lon': 0.95172, 'lat': 41.6566,
+                      'dist': 0.8583929293407604},
+                     {'id': 'C7', 'value': 5.6, 'altitude': 427,
+                      'lon': 1.16234, 'lat': 41.66695,
+                      'dist': 0.8387222708681318}]
+        
+        
         id3d = PyMica('id3d', 'pymica_tests/data/config_test.json')
-        field = id3d.interpolate('pymica_tests/data/init_data.json')
+        field = id3d.interpolate(data_dict)
```
Comment (ecasellas) [inline pymica_tests/pymica2_test.py:433]: The same here as the comment above.

File: pymica_tests/pymica2_test.py [hunk 3]
```diff
@@ -432,13 +451,20 @@ def test_init_interpolate_mlr(self):
...
+        data_dict = [{'id': 'C6', 'value': 6.4, 'altitude': 264,
+                      'lon': 0.95172, 'lat': 41.6566,
+                      'dist': 0.8583929293407604},
+                     {'id': 'C7', 'value': 5.6, 'altitude': 427,
+                      'lon': 1.16234, 'lat': 41.66695,
+                      'dist': 0.8387222708681318}]
+
         mlr = PyMica('mlr', 'pymica_tests/data/config_test.json')
-        field = mlr.interpolate('pymica_tests/data/init_data.json')
+        field = mlr.interpolate(data_dict)
```
Comment (ecasellas) [inline pymica_tests/pymica2_test.py:463]: The same, it seems that different data was used.. is it possible?

File: pymica_tests/pymica2_test.py (hunk 1 above)
Comment (ecasellas) [reply pymica_tests/pymica2_test.py:400]: I think that the code is still ok, but the data used to interpolate should be loaded from the same json file, and then pass just de dictionary you get.

**Review by ecasellas — CHANGES_REQUESTED** [review]

File: pymica_tests/pymica2_test.py (hunk 1 above)
Comment (jrmiro) [reply pymica_tests/pymica2_test.py:400]: Solved

File: pymica_dev_jr.yml
Comment (jrmiro) [reply pymica_dev_jr.yml:1]: Deleted

File: pymica_tests/pymica2_test.py (hunk 2 above)
Comment (jrmiro) [reply pymica_tests/pymica2_test.py:433]: Solved

**Review by ecasellas — APPROVED** [review]
